doc/source/reference/ufuncs.rst

```diff
@@ -327,6 +327,14 @@ advanced usage and will not typically be used.
     multiple outputs is deprecated, and will raise a warning in numpy 1.10,
     and an error in a future release.
 
+    If out is missing or None, a return value is created with uninitialized
+    values.  The `ufunc` is applied, if `where` is used the `ufunc` will be
+    applied where `where` is `True`. Note that the return value may have
```

Comment:
Maybe "The output array is then filled with the results of the `ufunc` in the places that `where` is true, by default everywhere if `where` is not specified. Note that outputs not explicitly filled are left with their uninitialized values."

Reply:
adopted